src/main/java/com/wiley/elements/custom/Style.java

```diff
@@ -0,0 +1,8 @@
+package com.wiley.elements.custom;
+
+public interface Style {
```

Comment:
maybe this should be more general "Attribube"? i mean  if i look at getStyle and pass... lets say "href" as an attribute value? - what is the expexted behavior? (lets discuss)